sites/footers/reports/a11y-1.json

```diff
@@ -0,0 +1,905 @@
+{
+  "@context": "https://wet-boew.github.io/GCWeb/sites/footers/index.json-ld",
+
+  "@id": "https://wet-boew.github.io/GCWeb/sites/footers/reports/a11y-1.json",
+  "id": "1",
+  "@type": "earl:Assertion",
+  "earl:test": ["WCAG20", "WCAG21", "WCAG22"],
+
+  "subjectUrl": "https://universallabs.org/wet/labs/pr2043/sites/footers/footers-en.html",
+
+  "assertedBy": {
+    "name": "Service Canada - Principal Publisher",
+    "@id": "https://github.com/ServiceCanada",
+    "@type": ["earl:Assertor", "foaf:Organization"],
+    "producedBy": {
+      "name": "Brahim Mahadi Wachilli (Github: @BrahimMahadi)",
+      "@id": "https://github.com/BrahimMahadi",
+      "@type": ["earl:Assertor", "foaf:Person"]
+    }
+  },
+  "date": "2022-12-02",
+
+  "assessment": "Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2",
+  "notes": "",
+  "score": "100%",
+
+  "review": [
+    {
+      "name": "Pierre Dubois (Github: @duboisp)",
+      "page": "https://github.com/duboisp",
+      "@type": ["wb-arc:Revision", "foaf:Person"],
+      "earl:mode": "earl:manual",
+      "earl:outcome": "earl:untested",
+      "outcome": "Untested",
+      "notes": "",
+      "date": "2022-09-22"
+    }
+  ],
+
+  "results": [
+    {
+      "label": "SC 1.1.1 Non-text content (Level A)",
+      "earl:test": "WCAG20:non-text-content",
+      "outcome": "Pass",
+      "earl:outcome": "earl:passed",
+      "notes": "",
+      "techniques": [
+        "WCAG21Tech:general/G94",
+        "WCAG21Tech:general/G95",
+        "WCAG21Tech:general/G82",
+        "WCAG21Tech:failures/F3",
+        "WCAG21Tech:failures/F13",
+        "WCAG21Tech:failures/F20"
+      ],
+      "earl:mode": "earl:automatic",
+      "@type": ["earl:TestResult", "earl:Assertion"]
+    },
+    {
+      "label": "SC 1.2.1 Audio-only and Video-only (Prerecorded) (Level A)",
+      "earl:test": "WCAG20:audio-only-and-video-only-prerecorded",
+      "outcome": "Pass",
```

Comment:
This is not applicable as the footer do not contains any audio nor video content. Idem for the other SC related to audio and video

Reply:
As discussed with Brahim, this is what the automated tool has reported. Let's keep it as is.